<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.lohika.gap.tests.LoginTest" tests="5" time="5.716" ignored="28" failures="2" timestamp="28 бер. 2017 15:07:25 GMT" skipped="3" hostname="vova-pc" errors="0">
  <testcase name="testSignin" time="0.000" classname="com.lohika.gap.tests.LoginTest">
    <skipped/>
  </testcase> <!-- testSignin -->
  <testcase name="testSignin_IncorrectCredentials" time="0.000" classname="com.lohika.gap.tests.LoginTest">
    <skipped/>
  </testcase> <!-- testSignin_IncorrectCredentials -->
  <testcase name="testSignin_IncorrectCredentials" time="0.000" classname="com.lohika.gap.tests.LoginTest">
    <skipped/>
  </testcase> <!-- testSignin_IncorrectCredentials -->
  <testcase name="testSignout" time="3.034" classname="com.lohika.gap.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="User is not signed out expected [The world&#039;s leading software development platform В· GitHub] but found [The world&#039;s leading software development platform · GitHub]">
      <![CDATA[java.lang.AssertionError: User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.lohika.gap.tests.LoginTest.testSignout(LoginTest.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </failure>
  </testcase> <!-- testSignout -->
  <testcase name="testSignout" time="2.682" classname="com.lohika.gap.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="User is not signed out expected [The world&#039;s leading software development platform В· GitHub] but found [The world&#039;s leading software development platform · GitHub]">
      <![CDATA[java.lang.AssertionError: User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.lohika.gap.tests.LoginTest.testSignout(LoginTest.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </failure>
  </testcase> <!-- testSignout -->
  <testcase name="testCreateNewRepository" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testCreateNewRepository -->
  <testcase name="testCreateNewRepository_NoName" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testCreateNewRepository_NoName -->
  <testcase name="testRenameRepository_NameAlreadyExist" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testRenameRepository_NameAlreadyExist -->
  <testcase name="testRemoveRepository" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testRemoveRepository -->
  <testcase name="testRenameRepository" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testRenameRepository -->
  <testcase name="testCreateNewRepository_AlreadyExist" classname="com.lohika.gap.tests.ManageRepositoriesTest">
    <skipped/>
  </testcase> <!-- testCreateNewRepository_AlreadyExist -->
  <testcase name="testRemoveCurrentDescription" classname="com.lohika.gap.tests.RepositoryDescriptionTest">
    <skipped/>
  </testcase> <!-- testRemoveCurrentDescription -->
  <testcase name="testSetNewDescription" classname="com.lohika.gap.tests.RepositoryDescriptionTest">
    <skipped/>
  </testcase> <!-- testSetNewDescription -->
  <testcase name="testSetNewDescription_Canceled" classname="com.lohika.gap.tests.RepositoryDescriptionTest">
    <skipped/>
  </testcase> <!-- testSetNewDescription_Canceled -->
  <testcase name="testSetNewDescription_IncorectWebpage" classname="com.lohika.gap.tests.RepositoryDescriptionTest">
    <skipped/>
  </testcase> <!-- testSetNewDescription_IncorectWebpage -->
  <testcase name="testSignin" classname="com.lohika.gap.tests.LoginTest">
    <skipped/>
  </testcase> <!-- testSignin -->
  <testcase name="testAddNewFile_Canceled" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testAddNewFile_Canceled -->
  <testcase name="testAddNewFile_AlreadyExists" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testAddNewFile_AlreadyExists -->
  <testcase name="testUploadFile" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testUploadFile -->
  <testcase name="testRemoveFile" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testRemoveFile -->
  <testcase name="testEditFile" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testEditFile -->
  <testcase name="testUploadFileViaAUTOIT" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testUploadFileViaAUTOIT -->
  <testcase name="testUploadFile_Failed" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testUploadFile_Failed -->
  <testcase name="testAddNewFile" classname="com.lohika.gap.tests.RepositoryFilesTest">
    <skipped/>
  </testcase> <!-- testAddNewFile -->
  <testcase name="testRemoveEmail" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testRemoveEmail -->
  <testcase name="testAddNewEmail" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testAddNewEmail -->
  <testcase name="testChangePassword_WrongOldPassword" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testChangePassword_WrongOldPassword -->
  <testcase name="testChangeEmailPreferences" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testChangeEmailPreferences -->
  <testcase name="testChangeUserProfile" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testChangeUserProfile -->
  <testcase name="testChangePassword_WrongNewPassword" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testChangePassword_WrongNewPassword -->
  <testcase name="testOpenGooglePage" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testOpenGooglePage -->
  <testcase name="testOpenGooglePage" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testOpenGooglePage -->
  <testcase name="testAddNewEmail_AlreadyExist" classname="com.lohika.gap.tests.ChangeSettingsTest">
    <skipped/>
  </testcase> <!-- testAddNewEmail_AlreadyExist -->
</testsuite> <!-- com.lohika.gap.tests.LoginTest -->
